composer.json

```diff
@@ -1,31 +1,43 @@
 {
   "name": "openconext/monitor-bundle",
   "type": "symfony-bundle",
-  "description": "A Symfony 4 bundle that facilitates health and info endpoints to a Symfony application. The bundle is backwards compatible with Symfony 2 projects.",
+  "description": "A Symfony 4/5/6 bundle that facilitates health and info endpoints to a Symfony application. The bundle is backwards compatible with Symfony 2 projects.",
   "keywords": ["SURFnet", "StepUp", "OpenConext", "monitoring", "health"],
   "license": "Apache-2.0",
   "minimum-stability": "stable",
   "require": {
-    "php": ">=5.4,<8.0-dev",
-    "symfony/dependency-injection": ">=3.4,<5",
-    "symfony/framework-bundle": ">=3.4,<5",
-    "webmozart/assert": "^1.2"
+    "php": ">=7.2",
```

Comment:
Perhaps add a <9.0-dev restriction. I feel this added defense will prevent some installation issues in the future.

Reply:
i am not sure if this is correct. "php": ">=7.2, <9.0-dev"
Can you please check it

Reply:
My Composer had no issue with `"php": ">=5.4,<9.0-dev" Did you run into issues while adding that additional constraint?